### ISSUE TITLE
The loss is hardly to reduce and the mAP is always zero

### ISSUE THREAD

**Issue by yololo1** [rodrigo2019/keras-yolo2#14]
I use the master repo
and i find the loss is hardly to reduce and the mAP is always zero
I use the raccoon dataset

**Comment by rodrigo2019** [comment]
how many epochs did you trained?

**Comment by yololo1** [comment]
> how many epochs did you trained?

153.
I found that，in epoch 57，the map is 0 and the loss is 10.22
in epoch 74 ，the map is 0.2 and the loss is 9.02
finally，in epoch 153  i only get a  0.41 map and the loss is 7.56
I found the new keras-loss is hard to train，is it？

this is my config， i use squeezenet
{
    "model" : {
        "backend":              "SqueezeNet",
        "input_size_w":         224,
        "input_size_h":         224,
        "gray_mode":            false,
        "anchors":              [5.68744,8.19239, 8.66723,14.83145, 10.63554,11.19373, 13.89734,16.22634, 14.38791,12.74825],
        "max_box_per_image":    10,        
        "labels":               ["raccoon"]
    },

    "parser_annotation_type":    "xml",

    "train": {
        "train_csv_file":       "",
        "train_csv_base_path":  "",
        "train_image_folder":   "/Users/keras_yolo2/raccoon_dataset-master/images",
        "train_annot_folder":   "/Users/keras_yolo2/raccoon_dataset-master/annotations",     

        "callback":             null,
        "train_times":          1,
        "pretrained_weights":   "",
        "batch_size":           4,
        "learning_rate":        1e-4,
        "nb_epochs":            150,
        "warmup_epochs":        3,

        "workers":              12,
        "max_queue_size":       40,
        "early_stop":           false,
        "tensorboard_log_dir":  "./logs/1",

        "object_scale":         5.0 ,
        "no_object_scale":      1.0,
        "coord_scale":          1.0,
        "class_scale":          1.0,

        "saved_weights_name":   "./SuperTinyYolo_plateArea32x32.h5",
        "debug":                false
    },

    "valid": {
        "iou_threshold":        0.5,
        "score_threshold":      0.5,
        "valid_csv_file":       "",
        "valid_csv_base_path":  "",
        "valid_image_folder":   "",
        "valid_annot_folder":   "",

        "valid_times":          1
    },

    "backup":{
        "create_backup":        false,
        "redirect_model":       false,
        "backup_path":          "../backup",
        "backup_prefix":        "Tiny_yolo_VOC"
    }
}

Thank you，best regards！

**Comment by rodrigo2019** [comment]
Didi you comapred your training with the experiencor repository?
